tests/integration/nupic/opf/hotgym_regression_test.py

```diff
@@ -61,10 +60,10 @@ def testHotgymRegression(self):
                          "steps=1:window=1000:field=consumption")
         lastRow = collections.deque(reader, 1)[0]
 
-      # Changes that affect prediction results will cause this test to fail.
-      # If the change is understood and reviewers agree that there has not been a
+      # Changes that affect prediction results will cause this test to fail. If
+      # the change is understood and reviewers agree that there has not been a
       # regression then this value can be updated to reflect the new result.
-      self.assertAlmostEqual(float(lastRow[14]), 5.90725098207)
+      self.assertAlmostEqual(float(lastRow[14]), 5.87970891453)
```

Comment:
@subutai, according to @rcrowder's comment https://github.com/numenta/nupic/pull/2952#issuecomment-172668787, the test should pass without this assertion change with a more recent nupic core.